### PR TITLE
Add form UI

### PR DESCRIPTION
**Pull request by toshiakitakase**
Only the UI of the form is implemented.

### CODE REVIEW

**Comment by wild-mouse** [comment]
Text color of optional checkboxes is similar to background color, so they are almost unreadable.
Screen Shot 2020-10-12 at 12 01 49

**Comment by toshiakitakase** [comment]
I'll get the layout and color adjustments together and do it later!

File: frontend/packages/react-app/src/components/organisms/CreateTokenForm.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <NumberInput
+              label="Team"
+              handleInput={teamInput}
+              inputValue={teamValue}
+              placeholder="30%"
+            />
+            <NumberInput
+              label="Community"
+              handleInput={communityInput}
+              inputValue={communityValue}
```
Comment (wild-mouse) [inline frontend/packages/react-app/src/components/organisms/CreateTokenForm.jsx:89]: Could you calculate community allocation with inputted `teamValue`? 
Contract constructor use only either one allocation value(contract argument name is ambiguous, I'm gonna update more explicitly
https://github.com/TART-tokyo/fan-stock/blob/f90d85a42be81f093f497dc6806be5541a44c1b9/contract/contracts/TokenFactory.sol#L37 ).


Simply update field `readonly` and subtract `teamValue` from `100` here is enough now.

**Review by wild-mouse — APPROVED** [review]
LGTM other than comments 🏃

File: frontend/packages/react-app/src/components/organisms/CreateTokenForm.jsx (hunk 1 above)
Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/organisms/CreateTokenForm.jsx:89]: Ok!

**Pull request merged**